CHANGELOG.md

```diff
@@ -24,6 +24,7 @@ This changelog keeps track of work items that have been completed and are ready
 - **General**: Automatically tag Docker image with commit SHA ([#567](https://github.com/kedacore/http-add-on/issues/567))
 - **RBAC**: Introduce fine-grained permissions per component and reduce required permissions ([#612](https://github.com/kedacore/http-add-on/issues/612))
 - **Operator**: Migrate project to Kubebuilder v3 ([#625](https://github.com/kedacore/http-add-on/issues/625))
+- **Routing**: New HTTPSO-based Routing Table ([#605](https://github.com/kedacore/http-add-on/issues/605))
```

Comment:
Let's be a bit more concrete here for our end-users - What does this exactly mean for them?